### PR TITLE
Add eol to word regex

### PR DESCRIPTION
**Pull request by dsandstrom**
I added the EOL symbol as an option for the word regex.  This will be used in atom/vim-mode#581.  Let me know if tests should be added.

### CODE REVIEW

File: src/cursor.coffee [hunk 1]
```diff
@@ -641,15 +641,20 @@ class Cursor extends Model
   # * `options` (optional) {Object} with the following keys:
   #   * `includeNonWordCharacters` A {Boolean} indicating whether to include
   #     non-word characters in the regex. (default: true)
+  #   * `includeEol` A {Boolean} indicating whether to include the end of the
```
Comment (nathansobo) [inline src/cursor.coffee]: Can we call the option `includeEOL`? We standardized on all caps for initialisms for 1.0.

**Comment by nathansobo** [comment]
Even a test that called the method with the option and tested the regex would be nice, just to make sure this doesn't regress in core.

**Comment by dsandstrom** [comment]
For some reason, whenever I rebase a PR, I end up with double commits.  I'll submit a new one.

**Comment by nathansobo** [comment]
Should we close this one then? Let me know when you've submitted the new one and correct the capitalization of the option.

**Pull request closed without merging**